site-src/implementations.md

```diff
@@ -105,6 +107,36 @@ In this section you will find specific links to blog posts, documentation and ot
 [epicdocs]:https://www.epic-gateway.org/
 [epicsource]:https://github.com/epic-gateway
 
+### Airlock Microgateway
+[![Conformance](https://img.shields.io/badge/Gateway%20API%20Conformance%20v1.1.0-Airlock%20Microgateway-green)](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.1.0/airlock-microgateway)
+
+[Airlock Microgateway][airlock-microgateway] is a Kubernetes native WAAP (Web Application and API Protection) solution to protect microservices.
+Modern application security is embedded in the development workflow and follows DevSecOps paradigms.
+Airlock Microgateway protects your applications and microservices with the tried-and-tested Airlock security features against attacks, while also providing a high degree of scalability.
+
+With [Airlock Microgateway 4.4][airlock-microgateway-v4.4], Airlock Microgateway introduces a sidecarless data plane mode
+based on Gateway API to avoid the operational complexity of sidecars.
+
+#### Features
+- Kubernetes native integration with sidecar injection and Gateway API support
+- Reverse proxy functionality with request routing rules, TLS termination and remote IP extraction
+- Using native Envoy HTTP filters like Lua scripting, RBAC, ext_authz, JWT authentication
+- Content security filters for protecting against known attacks (OWASP Top 10)
+- API security features like JSON parsing, OpenAPI specification enforcement or GraphQL schema validation
+
+#### Documentation and links
+- [Product documentation][airlock-microgateway-documentation]
+- [Gateway specific documentation][airlock-microgateway-guide]
+- Check our [Airlock community forum][airlock-microgateway-community-support] for support
+- Extended support is available for paid licenses following [the premium support process][airlock-microgateway-premium-support]
```

Comment:
It's great you are adding links about support, this is something very useful for users. We do have a request for you though: since Gateway API is an open-source project, we try to stay away when it comes to licenses and paid support. We'd like to ask you to rephrase these sentences by not mentioning "extended support", "paid license", or "premium support". We are fine with leaving the links here, but just be sure you refer to support in a generic way, without making distinctions between the community and EE versions.